lib/rules/no-wait-for-side-effects.ts

```diff
@@ -76,8 +86,32 @@ export default createTestingLibraryRule<Options, MessageIds>({
       }
     }
 
+    function reportImplicitReturnSideEffect(node: TSESTree.CallExpression) {
+      if (!isCallerWaitFor(node)) {
+        return;
```

Comment:
What's the difference between breaking out of the function and returning `false`? 🤔

Reply:
I'd say there is no difference? It's just a way to stop the lining as soon as you know it won't be reported, but I guess we could stick to just `return;` to make it more consistent.

Reply:
That's unnecessary. Leftover from my part 😶

Reply:
Fixed in b08d377